Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.yml

```
@@ -0,0 +1,5907 @@
+category: Case Management
+commonfields:
+  id: TOPdesk
+  version: -1
+configuration:
+- display: Server URL (e.g., https://topdesk.mydomain/tas/)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: username
+  required: true
+  type: 0
+- display: Password
+  name: password
+  required: true
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: true
+  display: Always use new style FIQL query (e.g. query=status==firsLine)
+  additionalinfo: Getting incidents with new style FIQL query is available from TOPDeskRestApi version 3.3.0. If the checkbox is unchecked, old style queries will be used.
+  name: new_query
+  required: false
+  type: 8
+- defaultvalue: '10'
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- defaultvalue: 3 days
+  display: First fetch time (<number> <time unit>, e.g., 12 hours, 7 days, 3 months,
+    1 year)
+  name: first_fetch
+  required: false
+  type: 0
+- display: The query to use when fetching incidents
+  name: fetch_query
+  required: false
+  type: 0
+- display: The earliest modification date to use when fetching incidents (<yyyy-mm-dd>
+    e.g. 2020-01-01)
+  name: modification_date_start
+  required: false
+  type: 0
+- display: The latest modification date to use when fetching incidents (<yyyy-mm-dd>
+    e.g. 2020-01-03)
+  name: modification_date_end
+  required: false
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: TOPdesk’s Enterprise Service Management software (ESM) lets your service
+  teams join forces and process requests from a single platform.
+display: TOPdesk
+name: TOPdesk
+script:
+  commands:
+  - arguments: []
+    deprecated: false
+    description: Get list of subcategories.
+    name: topdesk-subcategories-list
+    outputs:
+    - contextPath: TOPdesk.subcategory..id
+      description: ''
+      type: String
+    - contextPath: TOPdesk.subcategory..name
+      description: ''
+      type: String
+    - contextPath: TOPdesk.subcategory..category.id
+      description: ''
+      type: String
+    - contextPath: TOPdesk.subcategory..category.name
+      description: ''
+      type: String
+  - arguments: []
+    deprecated: false
+    description: Get list of categories.
+    name: topdesk-categories-list
+    outputs:
+    - contextPath: TOPdesk.category..id
```

```suggestion
    - contextPath: TOPdesk.category.id
```
not sure how you got this double dot situation -but please make sure you are using only one dot between context levels.